src/server/license/server/api_server.go

```diff
@@ -109,7 +109,8 @@ func (a *apiServer) activate(ctx context.Context, req *lc.ActivateRequest) (resp
 	// Allow request to override expiration in the activation code, for testing
 	if req.Expires != nil {
 		customExpiration, err := types.TimestampFromProto(req.Expires)
-		if err == nil && expiration.After(customExpiration) {
+		// TODO(fahad): Determine if the call to expiration.After() can be safely removed.
+		if err == nil /* && expiration.After(customExpiration) */ {
```

Comment:
Why does this check need to be removed?. It prevents the expiration time from being overridden to an later value